### PR TITLE
feat: switch to coredb

### PR DESCRIPTION
**Pull request by shaarful**
## Description
Now there are two login systems. one from oidc another the bahis serve default login. there will no conflict with dual login system because the user table is same for both server

## Dependency Changes

add requests-oauthlib library. I have tested for version 0.8.0.
python-dotenv psycopg2

## Database Changes

Need to reset previous Oauth Provider migration by running `python manage.py migrate oauth2_provider zero` from bahisweb container.
Then migrate the latest by running `python manage.py migrate oauth2_provider` from bahis-oidc container.

## Checklist

- [x] I have read the road86 Contribution Guide.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] My changes generate no new warnings.
- [x] I have made corresponding changes to the documentation.
- [x] New database changes have been committed.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SECRET_KEY='%s@@0ml3qt&%ge3sl=+^f269qdu0x3v5u5idw(=10cf$scqk=@'
+
+PSQL_DB_HOST="pgdb"
+PSQL_DB_PORT=5432
+#PSQL_DB_HOST="localhost"
```
Comment (ChasNelson1990) [inline .env:5]: Should these go in a .env.local file?

File: .env [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+#PSQL_DB_PORT=99
+PSQL_DB_NAME='coredb'
+PSQL_DB_USER='kobo'
+PSQL_DB_PASS='bahis@321'
+
+DEBUG=True
```
Comment (ChasNelson1990) [inline .env:11]: Should only be true for local right, not when deployed?

File: .env [hunk 3]
```diff
@@ -0,0 +1,13 @@
+SECRET_KEY='%s@@0ml3qt&%ge3sl=+^f269qdu0x3v5u5idw(=10cf$scqk=@'
```
Comment (ChasNelson1990) [inline .env:1]: Set to `"DJANGO_SECRET_KEY"`

File: .gitignore [hunk 4]
```diff
@@ -3,8 +3,9 @@
...
-db.sqlite3
-db.sqlite3-journal
+*.sqlite3
+*.sqlite3-journal
 media
 .idea/
-static/
\ No newline at end of file
+static/
+/.env
```
Comment (ChasNelson1990) [inline .gitignore:11]: You shouldn't need the `/` at the start here

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
A few minor changes just for clarity but otherwise looks good.

File: .env (hunk 1 above)
Comment (shaarful) [reply .env:5]: I want to remove the `.env` file from repo. I have added it git ignore file but somehow it pushed.

**Pull request merged**